raiden/ui/cli.py

```diff
@@ -126,6 +126,19 @@
         type=str,
     ),
     click.option(
+        '--database-dir',
```

Comment:
Can you rename this option to `--datadir` now since it encompasses all raiden data and not just the WAL database?